apps/condo/domains/user/schema/UserSudoToken.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Generated by `createschema user.UserSudoToken 'token:Text; expiresAt:DateTimeUtc; user:Relationship:User:CASCADE; remainingUses:Integer'`
+ */
+
+const { GQLError, GQLErrorCode: { INTERNAL_ERROR, BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/user/access/UserSudoToken')
+const { MAX_NUMBER_OF_TOKEN_USES } = require('@condo/domains/user/constants/sudoToken')
+
+
+const ERRORS = {
+    REMAINING_USES_INVALID: {
+        variable: ['data', 'remainingUses'],
+        code: INTERNAL_ERROR,
```

Comment:
BAD_USER_INPUT ?

Reply:
This field is controlled only by us, so I chose internal error